### ISSUE TITLE
False positive: any`at` property of object detected as `ArrayPrototypeAt` when using `--checkFeatures`

### ISSUE THREAD

**Issue by ItMaga** [yowainwright/es-check#387]
Code to reproduce:

```js
//test-array-at.js
const foo = {
	at() {
	}
};

foo.at();
```

Command:

```bash
npx es-check es12 "./test-array-at.js" --checkFeatures
```

**Comment by ItMaga** [comment]
@yowainwright Thank you!

Seems like it's not fully resolved. There is an issue with params

```js
const foo = {
	at() {
	}
};

foo.at(); // ✅ passes
foo.at(-1) // ❌ fails

```
